pq-src/Cargo.toml

```diff
@@ -22,17 +22,13 @@ keywords = ["bindings", "libpq", "pq-sys", "bundled"]
 categories = ["database", "external-ffi-bindings"]
 readme = "README.md"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [target.'cfg(not(target_os = "windows"))'.dependencies]
-openssl-sys = { version = "0.9.93", features = ["vendored"] }
+openssl-sys = "0.9.93"
 
 [build-dependencies]
 cc = "1.0.83"
 
-[target.'cfg(not(target_os = "windows"))'.build-dependencies]
```

Comment:
This dependency is not necessary, it is enabled with the `vendored` feature of `openssl-sys`.

Reply:
The `DEP_OPENSSL_INCLUDE` variable is set by this line, defined in the build script of `openssl-sys`: https://github.com/sfackler/rust-openssl/blob/83b3186f5f0c82c28339c2b235b3bb4e5d324b7b/openssl-sys/build/main.rs#L108, so it is not required to add `openssl-src` as a dependency.

See the `DEP_<name>_<key>` section at https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts.

Reply:
Thanks for clarifying, I expected that to be set by `openssl-src`, but having it set in `openssl-sys` simplifies things obviously.